src/index.ts

```diff
@@ -24,14 +36,22 @@ export interface TransposeOptions {
 	 * containerQemuPath: Where to add the qemu binary on-container
 	 */
 	containerQemuPath: string;
+
+	/**
+	 * Optional file mode (permission) to assign to the Qemu executable,
+	 * e.g. 0o555. Useful on Windows, when Unix-like permissions are lost.
+	 */
+	qemuFileMode?: number;
 }
 
+interface Command extends Pick<parser.CommandEntry, 'name' | 'args'> {}
+
 type CommandTransposer = (
 	options: TransposeOptions,
-	command: parser.Command,
-) => parser.Command;
+	command: Command,
+) => Command;
```

Comment:
The original `parser.Command` type was coming from this repo's `typings/docker-file-parser.d.ts`, but the `docker-file-parser` repo itself now includes type information.